<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рукопашный бой - Система</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgba(230, 230, 250, 255), stop:0.5 rgba(135, 206, 235, 255), stop:1 rgba(70, 130, 180, 255));
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1131</width>
     <height>501</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    border: 1px solid black;
    border-bottom-color: #C2C7CB;
    border-top-left-radius: 1px;
    border-top-right-radius: 1px;
    min-width: 26ex;
    padding: 10px;
	font-family: Avenir Next;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: #b0c4de;
	font-family: Avenir Next;
}
 
QTabBar::tab:selected {
    border-bottom-color: #b0c4de;
	font-family: Avenir Next;
}
 
QTabBar::tab:!selected {
    margin-top: 0px;
	font-family: Avenir Next;
}</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_8">
    <property name="styleSheet">
     <string notr="true">#tab_8 {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Посещаемость</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label {
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Группа:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1071</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <property name="styleSheet">
     <string notr="true">#tab_7 {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Расписание занятий</string>
    </attribute>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>731</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2 {
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Группа:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>50</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>90</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>130</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_4 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>711</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">#tab_6 {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Тренеры</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_5 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_6 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_7 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1071</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>80</y>
       <width>298</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Поиск...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>1041</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <property name="styleSheet">
     <string notr="true">#tab_5 {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Спортсмены</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_8 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_9 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_10 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1071</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>80</y>
       <width>298</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Поиск...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>1041</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#tab {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Группы</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>782</x>
       <y>80</y>
       <width>298</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Поиск...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_11 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1071</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_12 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_13 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>1041</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <zorder>pushButton_11</zorder>
    <zorder>listView_4</zorder>
    <zorder>pushButton_12</zorder>
    <zorder>pushButton_13</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>textBrowser_4</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="styleSheet">
     <string notr="true">#tab_2 {
background-color: #e2f2fc;
border-radius:7%;
border: 1px solid black;
font-family: Avenir Next;
font-size: 16px;
}</string>
    </property>
    <attribute name="title">
     <string>Отчетность</string>
    </attribute>
    <widget class="QListView" name="listView_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>711</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_3 {
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Дата периода:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_4 {
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Группа:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumDate">
      <date>
       <year>2025</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>430</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_14 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>691</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>691</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>691</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1057</x>
     <y>0</y>
     <width>54</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton {
background-color: #cf7373;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
